go/framework/bertybridge/bridge_protocol.go

```diff
@@ -3,6 +3,7 @@ package bertybridge
 import (
 	"context"
 	"fmt"
+	"math/rand"
```

Comment:
```suggestion
	mrand "math/rand"
```

to follow the convention of the rest of the project